### ISSUE TITLE
Installation of bokeh>=2.0.0 adds `release` directory to site-packages.

### ISSUE THREAD

**Issue by eugene57** [bokeh/bokeh#9931]
I installed latest bokeh (2.0.1) with pip under python 3.7.4 (to a virtual environment).
I noticed that it added `release` directory to my `site-packages` directory. As a result `release` is treated as an importable package (while not registered on pypi).
Could you please advise if this is intended behaviour? I haven't found any mentions of this `release` package in bokeh docs. If it is intended, is it possible to move `release` under `bokeh` namespace to avoid naming clashes?
```
$ python -m venv /tmp/bokeh_venv
$ source /tmp/bokeh_venv/bin/activate
$ pip install -U pip setuptools
$ pip install bokeh
$ pip list | grep bokeh
bokeh             2.0.1  
$ ls ~/tmp/bokeh_venv/lib/python3.7/site-packages/ | grep release
release
$ pip show bokeh -f | grep release/
  release/__init__.py
  release/__main__.py
  release/build.py
  release/checks.py
  release/config.py
  release/credentials.py
  release/enums.py
  release/git.py
  release/publish.py
  release/stages.py
  release/system.py
  release/ui.py
```
This effect exists in all bokeh versions starting from 2.0.0.

**Comment by bryevdv** [comment]
Definitely not intended. I'm not even sure how that could happen.

**Comment by bryevdv** [comment]
I'm not sure there is not anything to "do" per se but I will see if there is something that can be automated either on the package creation side or on a post-check side.

**Comment by bryevdv** [comment]
closed by #9942
